src/tools/init.ts

```diff
@@ -0,0 +1,243 @@
+import { type InferSchema } from "xmcp"
+import { z } from "zod"
+import { markInitCalled } from "../_internal/global-state"
+
+export const schema = {
+  project_path: z
+    .string()
+    .optional()
+    .describe("Path to the Next.js project (defaults to current directory)"),
+}
+
+export const metadata = {
+  name: "init",
+  description: `⚠️ CALL THIS FIRST - Initialize Next.js DevTools MCP context and establish MANDATORY documentation requirements.
+
+**IMPORTANT: This tool MUST be called at the START of every Next.js development session.**
+
+This tool fetches the latest Next.js documentation and establishes ABSOLUTE requirements for using the nextjs_docs tool for ALL Next.js-related queries.
+
+Key Points:
+- Fetches latest Next.js LLM documentation from nextjs.org
+- Establishes MANDATORY requirement to use nextjs_docs for ALL Next.js concepts
+- Instructs AI to forget any prior Next.js knowledge and always query docs
+- Documents all available MCP tools (nextjs_docs, nextjs_runtime, browser_eval, upgrade_nextjs_16, enable_cache_components)
+
+Use this tool at the beginning of a Next.js session to:
+- Reset AI's Next.js knowledge baseline
+- Ensure 100% documentation-first approach with no exceptions
+- Understand available tools and their use cases
+- Follow best practices for Next.js development`,
+}
+
+export default async function init(args: InferSchema<typeof schema>): Promise<string> {
+  try {
+    const projectPath = args.project_path || process.cwd()
```

Comment:
prefer not give fallback which could cause confusion in the future, like which case is this for 
```suggestion
    const projectPath = args.project_path
```